packages/instantsearch.js/src/components/SearchBox/SearchBox.tsx

```diff
@@ -191,7 +192,7 @@ class SearchBox extends Component<
             rootProps={{
               className: cssClasses.submit,
               type: 'submit',
-              title: 'Submit the search query.',
+              title: 'Submit the search query',
```

Comment:
If we're really strict about it, this is breaking. A CSS rule targeting the element by `title` would no longer work.

Reply:
I don't think the prop changes on this PR should block us, as long as we provide consistent class names that are consistent and well documented, for customers to rely on.